opensearch/src/main/java/org/opensearch/sql/opensearch/request/ExecutableRequestBuilder.java

```diff
@@ -0,0 +1,11 @@
+package org.opensearch.sql.opensearch.request;
+
+import org.opensearch.sql.common.setting.Settings;
+
+public interface ExecutableRequestBuilder {
```

Comment:
Could you add JavaDoc for all new interfaces?

Reply:
Deleted `ExecutableRequestBuilder`  -- it was no longer required.